datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -7256,7 +7256,7 @@ input UpdateParentNodeInput {
     """
     The new parent node urn
     """
-    parentNode: String!
+    parentNode: String
```

Comment:
can we update this doc to say something like "if parentNode is null, this will remove the parent from this entity"?

Reply:
Done!